python/ray/llm/_internal/serve/core/ingress/ingress.py

```diff
@@ -30,6 +30,7 @@
 )
 from ray.llm._internal.serve.constants import (
     DEFAULT_LLM_ROUTER_HTTP_TIMEOUT,
+    DEFAULT_LLM_ROUTER_TARGET_ONGOING_REQUESTS,
```

Comment:
can we just use DEFAULT_MAX_TARGET_ONGOING_REQUESTS instead? I think we should remove the old names (LLM_ROUTER, etc) we just haven't had a chance to clean those up